### PR TITLE
Polex for default apps

### PR DESCRIPTION
**Pull request by njuettner**
Issue: https://github.com/giantswarm/giantswarm/issues/28345

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

**Comment by stone-z** [comment]
I think only cilium is needed to actually start the cluster, right?

File: helm/kyverno/templates/core-policies/cilium-policies-polex.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+kind: PolicyException
+metadata:
+  name: kyverno-app-cilium-policies-policy-exception
+  namespace: giantswarm
+  labels:
+    {{- include "kyverno-stack.labels" . | nindent 4 }}
+  annotations:
+    {{- include "kyverno-stack.policyInstallAnnotations" . | nindent 4 }}
+spec:
+  exceptions:
```
Comment (stone-z) [inline helm/kyverno/templates/core-policies/cilium-policies-polex.yaml:12]: That's a lot of exceptions :sweat_smile:

**Review by stone-z — APPROVED** [review]
Ok with me. I think we broke CI by accident in https://github.com/giantswarm/kyverno-app/pull/281 so don't worry about that. We'll sort it out

File: helm/kyverno/templates/core-policies/cilium-policies-polex.yaml (hunk 1 above)
Comment (njuettner) [reply helm/kyverno/templates/core-policies/cilium-policies-polex.yaml:12]: Yeah I know 😞 but that should be it now

**Comment by njuettner** [comment]
Ok that should be it for now @stone-z mind if you have a look again? It's super hard for me to detect violations for jobs because they are super fast, but other than that I tested it and don't see any issues for cilium-agent and cilium-operator.

File: helm/kyverno/templates/core-policies/cilium-policies-polex.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    ruleNames:
+    - app-armor
+    - autogen-app-armor
+  - policyName: disallow-privileged-containers
+    ruleNames:
+    - privileged-containers
+    - autogen-privileged-containers
+
+
+  match:
```
Comment (stone-z) [inline helm/kyverno/templates/core-policies/cilium-policies-polex.yaml:71]:
```suggestion
    - autogen-privileged-containers

  match:
```

File: helm/test.yaml [hunk 3]
```diff
@@ -0,0 +1,126 @@
+---
```
Comment (stone-z) [inline helm/test.yaml:1]: I think this file was committed by mistake?

**Review by stone-z — COMMENTED** [review]
Ok there's no way cilium actually needs all those exceptions to run the Job but I'll open a separate issue for that.
LGTM after the accidental file is removed

File: helm/test.yaml (hunk 3 above)
Comment (njuettner) [reply helm/test.yaml:1]: You're right 🤦🏻, this was used for testing the policy

**Comment by njuettner** [comment]
Regarding the policy for jobs, you're right we need to reduce the policy. For me it is just pretty hard to figure out what policy needs to be added. The job is deleted super fast and I cannot see the kyverno policy violation events 😞.

**Review by stone-z — APPROVED** [review]

**Pull request merged**